src/app/pages/server-compare/special-compares.js

```diff
@@ -0,0 +1,22 @@
+
+/*
+export interface SpecialCompare {
+    id: string;
+    name: string;
+    title: string;
+    description: string;
+    instances: any;
+}
+*/
+
+const specialCompares = [
+  {
+    id: 'test',
+    name: 'Test',
+    title: 'Test title',
+    description: 'Test dewscirption',
+    instances: 'W3sidmVuZG9yIjoiZ2NwIiwic2VydmVyIjoiZzEtc21hbGwifSx7InZlbmRvciI6ImdjcCIsInNlcnZlciI6Im4xLXN0YW5kYXJkLTEifV0'
```

Comment:
could this base64 encoded stuff calculated on the fly so that we define the list of instances in a JSON or similar more human-friendly (and readable) list?